data/initrd/initrd.file_list

```diff
@@ -435,10 +435,13 @@ openssh: nodeps
   E prein
   d etc/ssh
 
+system-group-hardware:
+  /
+  E prein
```

Comment:
it is called also in openssh and also dbus-1. So does we need to call it more times?

Reply:
it's the packages prein script

Reply:
ah, now I get what in documentation mean rpm script. So makes sense.